src/styles/blocks/card.scss

```diff
@@ -0,0 +1,129 @@
+.main {
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  justify-content: center;
+  padding: 50px 40px;
+  gap: 46px 48px;
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, 200px);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, 200px);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
+
+.card {
+  box-sizing: border-box;
+  width: $width;
+  border-radius: $border-radius;
+  border: $border;
+  font-family: Roboto, sans-serif;
+  transition: all 300ms;
+
+  &:hover {
+    transform: scale(1.2);
+
+    &__product-name {
+      color: #34568b;
+    }
+  }
+
+  &__container {
+    margin-right: 16px;
+    margin-left: 16px;
+  }
+
+  &__product-photo {
+    display: flex;
+    justify-content: center;
+  }
+
+  &__buy-button {
+    font-family: Roboto, sans-serif;
```

Comment:
You've already set this font value for the entire page, you don't need to set it again.